Assets/MRTK/Tests/PlayModeTests/SlateTests.cs

```diff
@@ -275,6 +478,38 @@ private void InstantiateFromPrefab(Vector3? position = null, Quaternion? rotatio
             panObject.transform.rotation = rotation != null ? (Quaternion)rotation : Quaternion.identity;
             panZoom = panObject.GetComponentInChildren<HandInteractionPanZoom>();
             Assert.IsNotNull(panZoom);
+            meshFilter = panZoom.GetComponent<MeshFilter>();
+        }
+
+        /// <summary>
+        /// Instantiates a slate from the default prefab at position and rotation and sets pan limit
+        /// </summary>
+        private void InstantiatePanLimitedSlateFromPrefab(Vector3? position = null, Quaternion? rotation = null,
+            float maxPanVertical = 1.0f, float maxPanHorizontal = 1.0f)
+        {
+            InstantiateFromPrefab(position, rotation);
+
+            var unlimitedPanField = typeof(HandInteractionPanZoom).GetField("unlimitedPan", BindingFlags.NonPublic | BindingFlags.Instance);
+            Assert.IsNotNull(unlimitedPanField);
+            unlimitedPanField.SetValue(panZoom, false);
+
+            var lockHorizontalField = typeof(HandInteractionPanZoom).GetField("lockHorizontal", BindingFlags.NonPublic | BindingFlags.Instance);
+            Assert.IsNotNull(lockHorizontalField);
+            lockHorizontalField.SetValue(panZoom, false);
+
+            var lockVerticalField = typeof(HandInteractionPanZoom).GetField("lockVertical", BindingFlags.NonPublic | BindingFlags.Instance);
+            Assert.IsNotNull(lockVerticalField);
+            lockVerticalField.SetValue(panZoom, false);
```

Comment:
I believe the default value for "lockvertical" and "lockhorizontal" is already false. Maybe this is not needed?

Reply:
The default value depends on the settings of the slate prefab.
I prefer to explicitly specify the value in the test code.